cloudconfig.go

```diff
@@ -11,11 +11,11 @@ import (
 )
 
 type CloudConfigTemplateParams struct {
-	Cluster   clustertpr.Cluster
-	Node      node.Node
-	TLSAssets CompactTLSAssets
-	Files     []FileAsset
-	Units     []UnitAsset
+	Cluster     clustertpr.Cluster
+	Node        node.Node
+	MasterFiles []FileAsset
+	WorkerFiles []FileAsset
```

Comment:
Why are we wanting to separate the files? As I see it the input is quite the same concept wise, no matter if it is executed against the master or the worker template. If there are differences, that I do not see, then the separation should be made towards the whole input. So there would be something like  `MasterCloudConfigTemplateParams` and `WorkerCloudConfigTemplateParams`. That would make sense in case we need to separate it, but this change made here feels like we are getting the abstraction wrong.